lib/cssom.js

```diff
@@ -58,5 +333,4 @@ declare class CSSStyleDeclaration {
   removeProperty(property: string): string;
   setProperty(property: string, value: ?string, priority: ?string): void;
   setPropertyPriority(property: string, priority: string): void;
-  setPropertyValue(property: string, value: string): void;
```

Comment:
Hmm...looking into it I see it in the (working draft)[http://dev.w3.org/csswg/cssom/#the-cssstyledeclaration-interface]. I don't feel too strongly, but I don't particularly mind leaving in future features.